src/us_reverse_geo/Client.js

```diff
@@ -0,0 +1,43 @@
+const Response = require("./SmartyResponse");
+const Request = require("../Request");
+const Lookup = require("./Lookup");
+const UndefinedLookupError = require("../Errors").UndefinedLookupError;
+const buildInputData = require("../util/buildInputData");
+const keyTranslationFormat = require("../util/apiToSDKKeyMap").usReverseGeo;
+
+/**
+ * This client sends lookups to the SmartyStreets US Street API, <br>
+ *     and attaches the results to the appropriate Lookup objects.
+ */
+class Client {
+	constructor(sender) {
+		this.sender = sender;
+	}
+
+	send(lookup) {
+		if (typeof lookup === "undefined") throw new Errors.UndefinedLookupError();
```

Comment:
You can new up `UndefinedLookupError()` directly since that is how you import it to this file
```suggestion
		if (typeof lookup === "undefined") throw new UndefinedLookupError();
```